ts/utils/payment.ts

```diff
@@ -35,7 +34,6 @@ import { getFullLocale, getLocalePrimaryWithFallback } from "./locale";
 import { maybeInnerProperty } from "./options";
 import { formatNumberCentsToAmount } from "./stringBuilder";
 import { maybeNotNullyString } from "./strings";
-import { getProfileDetailsLog } from "./profile";
```

Comment:
`getProfileDetailsLog` is not used anywhere else in the codebase, therefore can be safely removed.

Reply:
https://github.com/pagopa/io-app/pull/3362/commits/2e26de461a4cbf61e1c4ffa09ebc710278e46fb1